### PR TITLE
4.8 SRIOV release notes

### PR DESCRIPTION
**Pull request by mikemckiernan**
Preview link: https://deploy-preview-31430--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-networking

----
@openshift/team-documentation  PTAL.

Applies to enterprise-4.8 and milestone Future Release.

### CODE REVIEW

**Comment by mikemckiernan** [comment]
/assign @zshi-redhat 
/assign @zhaozhanqi 

Last one, folks.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -61,6 +61,16 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-networking"]
 === Networking
 
+[id="ocp-4-8-supported-hardware-sriov"]
+==== Supported hardware for SR-IOV
+
+{product-title} {product-version} adds support for additional Intel and Mellanox hardware.
+
+* Intel X710 controller
```
Comment (zshi-redhat) [inline release_notes/ocp-4-8-release-notes.adoc:69]: Once XL710 40G SFP+ is added, this will need to be updated.

**Comment by jboxman** [comment]
@mikemckiernan awesome, thanks!

**Comment by mikemckiernan** [comment]
/assign @zshi-redhat 

I added the XL710.  PTAL again and thank you.

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**